django/db/models/deletion.py

```diff
@@ -308,7 +312,9 @@ def delete(self):
             for model, instances in six.iteritems(self.data):
                 query = sql.DeleteQuery(model)
                 pk_list = [obj.pk for obj in instances]
-                query.delete_batch(pk_list, self.using)
+                count = query.delete_batch(pk_list, self.using)
+                name = '{0}.{1}'.format(model._meta.app_label, model._meta.object_name)
```

Comment:
Want to get [#24693](https://code.djangoproject.com/ticket/24693) fixed first so we can use it here and in the test, instead of adding that helper method.